Twaiq.playground/Contents.swift

```diff
@@ -0,0 +1,13 @@
+import UIKit
+import Foundation
+
+var Selectedcity = "Riadh"
+
+var cities = ["Medanah", "Makkah", "Jeddah", "Riadh", "Abha", "Taief", "Alqaseem", "Albaha", "Negran", "Jezan"]
+
+var Population = ["1,152,991", "1,578,722", "4,082,184", "7,677,000" /* ... */]
+
+if Selectedcity == cities[3] {
```

Comment:
You checked only the 4th element but we need complete check. if possible rewrite it using loop or in-built function or using dictionary as we discussed in the class.